packages/rn-tester-e2e/jest.setup.js

```diff
@@ -0,0 +1,31 @@
+const wdio = require('webdriverio');
+import capabilities from './e2e-config.js';
+import { beforeEach, afterEach, jest } from '@jest/globals';
+
+
+jest.setTimeout(40000);
+jest.retryTimes(3);
+
+let driver;
+const config = {
+  path: '/wd/hub',
+  host: 'localhost',
+  port: 4723,
+  waitforTimeout: 30000,
```

Comment:
yet another timeout with a different value from 60000 and 40000, let's just set the same one everywhere - or there's a reason why they are different?

Reply:
value has changed